tests/io/mongodb/test_cli.py

```diff
@@ -29,7 +33,39 @@ def test_version():
     assert exitcode == 0
 
 
-def test_mongodb_load_table(caplog, cratedb, mongodb):
+DOCUMENT_IN = {
+    "id": bson.Binary.from_uuid(UUID("d575540f-759d-4653-a4c4-4a9e410f1aa1")),
+    "value": {
+        "name": "foobar",
+        "active": True,
+        "created": dateutil.parser.parse("2020-06-19T15:03:53.727Z"),
+        "timestamp": bson.datetime_ms.DatetimeMS(1455141600000),
+    },
+}
+DOCUMENT_OUT = {
+    "__id": mock.ANY,
+    "id": "d575540f-759d-4653-a4c4-4a9e410f1aa1",
+    "value": {
+        "name": "foobar",
+        "active": True,
+        "created": 1592579033000,
+        "timestamp": 1455141600000,
+    },
+}
+DOCUMENT_DDL = """
+CREATE TABLE IF NOT EXISTS "testdrive"."demo" (
+   "__id" TEXT,
+   "id" TEXT,
+   "value" OBJECT(DYNAMIC) AS (
+      "name" TEXT,
+      "active" BOOLEAN,
+      "created" TIMESTAMP WITH TIME ZONE,
+      "timestamp" TIMESTAMP WITH TIME ZONE
+   )
+)""".lstrip()
```

Comment:
Best view into the integration test added with bf78869b3f943 and 249cac19ccd020394b.